test/website-docs.spec.js

```diff
@@ -51,4 +51,18 @@ describe('docs', () => {
   it('has headers for every required sub-option', () => {
     expect(headers3).toEqual(expectedOptions3);
   });
+
+  // Checking relatedOptions field in definitions
+  let relatedOptions = options
+    .filter(option => option.relatedOptions)
+    .map(option => option.relatedOptions)
+    .sort();
+
+  relatedOptions = [].concat(...relatedOptions);
+
+  const allOptionNames = options.map(option => option.name).sort();
+
+  it('has valid relateOptions values', () => {
+    expect(allOptionNames).toEqual(expect.arrayContaining(relatedOptions));
```

Comment:
I'm not familiar with this syntax. Is this the same as "expect relatedOptions to be a subset of allOptionNames" ?

Reply:
Is the error message meaningful to the user if they make a mistake? e.g. prints the incorrect option? Maybe a for loop and explicit error message would be better